README.md

```diff
@@ -1,2 +1,45 @@
 # MobilePortfolio
-Personal portfolio project
+Personal portfolio project for Microverse course (Module 1)
+
+The aim of this project is to learn to understand how to parse a Figma design, use flexbox to place elements in the page, build a personal portfolio site.
+
+## Built With
+
+- HTML
+- css
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] For better documentation please consider removing these unused headers from the README file